gsdk/tests/backtrace.rs

```diff
@@ -0,0 +1,45 @@
+// This file is part of Gear.
+//
+// Copyright (C) 2023 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use gsdk::{backtrace::BacktraceStatus, signer::Signer, Api, Result};
+use utils::{alice_account_id, dev_node, node_uri};
+
+mod utils;
+
+#[tokio::test]
+async fn transfer_backtrace() -> Result<()> {
```

Comment:
what about node logs?

Reply:
please open an issue to somehow redirect node log to be able to see during the test

Reply:
#3304